docs/rpc/state-sync.md

```diff
@@ -73,7 +75,7 @@ Check whether state sync is enabled, and check whether it's configured to get st
 If all seems to be configured fine, then disable state sync (set `"state_sync_enabled": false` in `config.json`) and try again.
 If that doesn't help, then restart from a backup data snapshot.
 
-## Running a Chunk-Only Producer that tracks a single shard
+## Running a validator that tracks a single shard
```

Comment:
This page is part of RPC node documentation but talks about Chunk-only producers. For now just changed the title, but we should either move the page to a higher layer (independent of node type) or remove this section.